examples/text-classification/run_pl_glue.py

```diff
@@ -36,6 +36,8 @@ def forward(self, **inputs):
         return self.model(**inputs)
 
     def training_step(self, batch, batch_idx):
+        self.lr_scheduler = self.trainer.lr_schedulers[0]["scheduler"]
```

Comment:
also here, I'm just concerned that lightning has already set it for us and we are overwriting.

Reply:
it was added because it hasn't already done so. There was no `self.lr_scheduler` available.

The initial PR was fixing a very broken `run_pl_glue` - as it had 5 errors (listed [here](https://github.com/huggingface/transformers/pull/6028), which wasn't keeping in sync with lightning_base changes. Luckily, we now have a test that will cover such future divergence.

I will take a look and see if it can be done in a cleaner way.

Reply:
OK, `self.lr_scheduler` isn't a thing in the examples, so I will just localize it